### DISCUSSION TITLE
`html_safe` not working?

### DISCUSSION THREAD

**Discussion by cmer** [yippee-fun/phlex#171] [Help] [answered]
I am trying to pass a raw HTML string as an argument like this:

```ruby
span "&nbsp;&nbsp;&centerdot;&nbsp;&nbsp;".html_safe
```

but it gets escaped and the browser outputs it as is. Is this a bug, expected, or is there another way around this?

**Comment by joeldrapper** [comment, marked as answer]
Hey @cmer, you can get unescaped output using the `raw` method. Sorry this isn’t documented at the moment.

```ruby
span { raw "&nbsp;&nbsp;&centerdot;&nbsp;&nbsp;" }
```

I wanted to avoid using `.html_safe` as it requires a String patch from Rails and it’s slower than passing the string to a `raw` method.

**Reply by cmer** [reply]
Thanks!
